Classes/NewComment/CommentFormatterAccessoryView.swift

```diff
@@ -0,0 +1,164 @@
+import Foundation
+import UIKit
+
+private struct MDownFormattingElement {
+    let name: String
+    let replacements: (String, String?) // before / after
+    let atLineStart: Bool
+
+    init(name: String, replacements: (String, String?)) {
+        self.name = name
+        self.replacements = replacements
+        self.atLineStart = false
+    }
+
+    init(name: String, replacements: (String, String?), atLineStart: Bool) {
+        self.name = name
+        self.replacements = replacements
+        self.atLineStart = atLineStart
+    }
+}
+
+private let baseElements = [
+    MDownFormattingElement(name: "mdown.codeBlock", replacements: ("```\n", "\n```")),
+    MDownFormattingElement(name: "mdown.code", replacements: ("`", "`")),
+    MDownFormattingElement(name: "H1", replacements: ("# ", nil), atLineStart: true),
+    MDownFormattingElement(name: "H2", replacements: ("## ", nil), atLineStart: true),
+    MDownFormattingElement(name: "H3", replacements: ("### ", nil), atLineStart: true),
+    MDownFormattingElement(name: "H4", replacements: ("#### ", nil), atLineStart: true),
+    MDownFormattingElement(name: "mdown.bold", replacements: ("**", "**")),
+    MDownFormattingElement(name: "mdown.emphasis", replacements: ("*", "*")),
+    MDownFormattingElement(name: "mdown.strike", replacements: ("~~", "~~"))
+]
+
+private func skinnedButton(_ button: UIButton) -> UIButton {
+    button.contentEdgeInsets = UIEdgeInsets(top: 5.0, left: 20.0, bottom: 5.0, right: 20.0)
+    button.backgroundColor = Styles.Colors.Blue.medium.color
+    button.layer.cornerRadius = 16.0 // just so we don't need recompute
+    button.layer.masksToBounds = true
+    button.setTitleColor(.white, for: .normal)
+    return button
+}
```

Comment:
I'll probably tweak these settings a little bit, not crazy about the blue background. Going to restyle to be more like the keyboard buttons